src/server/index.ts

```diff
@@ -68,6 +68,10 @@ checkEnvVariables(envVars);
 
 const port = getEnvVariable('PORT', envVars) as number;
 
+app.get('/ready', (_, res) => {
+  res.send('Faucet backend is ready.');
+});
+
```

Comment:
Doing this does not give us much more than the previous `/health` endpoint.
Here it would be great to have a success (200) response ONLY if we can query the API (any call).